### PR TITLE
[onecc-docker] onecc subtool error fix

### PR DESCRIPTION
**Pull request by holawan**
This PR corrected exception handling and subtool related errors.

ONE-DCO-1.0.-Signed-off-by: Dongwan Kim <asdf134652@gmail.com>

issue: https://github.com/Samsung/ONE/issues/9712

### CODE REVIEW

File: compiler/onecc-docker/onecc-docker [hunk 1]
```diff
@@ -51,7 +51,7 @@ def main():
         response = requests.get(LATEST_URL)
         response.raise_for_status()
     except requests.exceptions.RequestException as e:
-        SystemExit('onecc-docker: error: {}'.format(e))
+        raise SystemExit('onecc-docker: error: {}'.format(e))
```
Comment (holawan) [inline compiler/onecc-docker/onecc-docker:54]: https://github.com/Samsung/ONE/pull/9800#discussion_r980901883 

I checked our mistake and added `raise`.

File: compiler/onecc-docker/onecc-docker [hunk 2]
```diff
@@ -69,7 +69,7 @@ def main():
         print('Dockerfile successfully built.')
 
     contianer_name = f"onecc_{recent_version.replace('.','_')}"
-    user_cmd = ''.join(sys.argv[1:])
```
Comment (holawan) [inline compiler/onecc-docker/onecc-docker:72]: We have confirmed that the `onecc` subtools does not work in the conventional way. Adds a space.

File: compiler/onecc-docker/onecc-docker [hunk 3]
```diff
@@ -69,7 +69,7 @@ def main():
...
+    user_cmd = ' '.join(sys.argv[1:])
```
Comment (holawan) [inline compiler/onecc-docker/onecc-docker:72]: We have confirmed that the onecc subtools does not work in the conventional way. Adds a space.

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**